test/acceptance/features/bindAppToService.feature

```diff
@@ -215,7 +215,7 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "binding-request-empty-app" should be changed to "True"
         And Secret "binding-request-empty-app" contains "BACKEND_HOST" key with value "example.common"
         And Secret "binding-request-empty-app" contains "BACKEND_USERNAME" key with value "foo"
-
+    @disconnected
```

Comment:
we have already `@smoke` as the tag - why do we need another tag for disconnected cluster. i.e. what are scenario properties that make them suitable for testing in disconnected mode?